src/lib/Libwin/accesinfo.c

```diff
@@ -396,7 +395,6 @@ create_secure_dacl2(char *user, ACCESS_MASK mask, char *user2, ACCESS_MASK mask2
 	}
 	if (InitializeAcl(ndacl, cbAcl, ACL_REVISION) == 0) {
 		log_err(-1, __func__, "failed in InitializeAcl");
-		return NULL;
```

Comment:
removed return?

Reply:
Yes, added that by mistake. Actual code https://github.com/PBSPro/pbspro/blob/master/src/lib/Libwin/accesinfo.c#L386

Reply:
ohk got it